src/garage/tf/experiment/__init__.py

```diff
@@ -0,0 +1,4 @@
+"""TF Experiment functions."""
```

Comment:
No need to abbreviate TensorFlow here. we don't pay for code storage by the character :P